tests/initrd.c

```diff
@@ -4,6 +4,20 @@
 #include <mount.h>
 #include <vm_map.h>
 #include <ktest.h>
+#include <dirent.h>
+
+static void prepare_uio(uio_t *uio, iovec_t *iov, char *buffer, int buf_size) {
```

Comment:
If you're introducing such function, please make it globally accessible (#262) or don't use it at all. It multiple files bring their own version of uio setup routine, we'll end up with a confusing interface that is difficult to unify.